docs/typography.mdx

```diff
@@ -0,0 +1,156 @@
+---
+name: 'Typography'
+---
+
+<!-- workaround to drop swingset's styling of first h1 -->
+
+<h1 />
+
+# Typography
```

Comment:
Much of this content is ported over from `web-components` (it joins the former [typography](https://web-components-hashicorp.vercel.app/?path=/docs/base-theme-typography--page) and [g-type-*](https://web-components-hashicorp.vercel.app/?path=/story/packages-global-styles-components-g-type--demo) pages).